storage/InsertDestinationInterface.hpp

```diff
@@ -104,7 +104,7 @@ class InsertDestinationInterface {
    *        insertion from ValueAccessor even when partially full.
    **/
   virtual void bulkInsertTuples(ValueAccessor *accessor,
-                                bool always_mark_full = false) = 0;
+                                const bool always_mark_full = false) = 0;
```

Comment:
Similar to other variants of bulkInsert methods, why are we not getting rid of the ``always_mark_full`` parameter in this function?

Reply:
It is required by the first pass of the multi-pass sort.